cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -1575,15 +1580,11 @@ struct DispatchRadixSort :
         typedef typename DispatchRadixSort::MaxPolicy       MaxPolicyT;
         typedef typename ActivePolicyT::SingleTilePolicy    SingleTilePolicyT;
 
-        // Return if empty problem
-        if (num_items == 0)
+        // Return if empty problem, or if no bits to sort and double-buffering is used
+        if (num_items == 0 || begin_bit == end_bit && is_overwrite_okay)
         {
-          if (d_temp_storage == nullptr)
-          {
-            temp_storage_bytes = 1;
-          }
-
-          return cudaSuccess;
+            if (d_temp_storage == nullptr) temp_storage_bytes = 1;
```

Comment:
Leave the explicit braces / linebreaks in the body here -- we're in the process of migrating code to use the safer style and want the body explicitly bounded.

Reply:
Done.